test/e2e/externalservices/localityawarelb_multizone/locality_multizone_hybrid.go

```diff
@@ -312,7 +140,7 @@ func ExternalServicesOnMultizoneHybridWithLocalityAwareLb() {
 		Expect(zone1.(*K8sCluster).StopZoneEgress()).To(Succeed())
 
 		// then service is unreachable
-		_, _, err := zone4.ExecWithRetries("", "", "zone4-demo-client",
+		_, _, err := zone4.ExecWithCustomRetries("", "", "zone4-demo-client", 3, time.Second*3,
```

Comment:
since it's the one test suite, the first test will stop ZoneEgress, then the second one will stop ZoneIngress, but ZoneEgress will be already stopped. No?

Reply:
Ok, this test is weird. We are testing if something fails, but we use `ExecWithCustomRetries` that actually execute something and waits to not fail.

I think the test should be
* Wait that traffic is OK
* Stop Zone Ingress
* Eventually traffic fails

Right now it is:
* Stop Zone Ingress
* Do retries until the traffic is ok (but never is, nor it should be)
* Check if it failed